apps/directory/src/views/ConfigurationScreen.vue

```diff
@@ -0,0 +1,380 @@
+  applyToColumn: "myColumn"                                  /** the column in the main table to apply the filter on.                                          */
+  filterValueAttribute:"id"                                  /** specify a column name if you want a different column for the value                            */
+  filterLabelAttribute: "name"                               /** specify if you want to use another column as the label for the filter option, instead of name */
+  ontologyIdentifiers: []                                    /** for use when you have multiple ontologies in a single table, e.g. orhpa and icd               */
+  trueOption: { text: "When clicked is true", value: true }, /** use this for a togglefilter                                                                   */
+  matchTypeForFilter: "any"                                  /** if it has been selected from bookmark, it will be applied here.                               */
+  showMatchTypeSelector: true                                /** if you want to make match all / match any available                                           */
+  negotiatorRequestString: "From my filterfacet: "           /** the part that will be send to the negotiator as to indicate what it is                        */
+  showFacet: true                                            /** if this filter should be shown at start                                                       */
+  adaptive: false                                            /** if the filters options should react on search results                                         */
+  sortColumn: "name"                                         /** specify a column to apply sorting on                                                          */
+  sortDirection: "asc"                                       /** the direction to sort                                                                         */
+}
+          </pre>
+        </small>
+      </div>
+    </div>
+
+    <!-- Advanced Editor -->
+    <json-editor
+      :config="currentConfig"
+      @save="saveFromEditor"
+      v-if="editorType === 'editor'"
+    />
+
+    <!-- Diff editor -->
+    <diff-editor
+      v-if="editorType === 'diff'"
+      :currentConfig="currentConfig"
+      :newConfig="uploadedAppConfig"
+      @save="saveFromEditor"
+      @cancel="switchView('editor')"
+    />
+
+    <!-- Landingpage editor -->
+    <div v-if="editorType === 'landingpage'" class="row px-5 pb-5">
+      <landingpage-editor
+        :currentConfig="currentConfig"
+        @save="saveLandingpage"
+        @cancel="switchView('editor')"
+      />
+    </div>
+    <!-- standard button bar -->
+    <div v-if="currentView === 'ui'" class="row px-5 pb-5">
+      <div class="col pl-0">
+        <button class="btn btn-primary mr-3 save-button" @click="save">
```

Comment:
The changes in ordering and names are saved correctly. But i don't see the changes reflected in the app.